src/palindrom.js

```diff
@@ -487,12 +487,20 @@ var Palindrom = (function() {
     reconnectionFn(function bootstrap(json) {
       palindrom.reconnector.stopReconnecting();
 
-      palindrom.queue.reset(palindrom.obj, json);
-
       if (palindrom.debug) {
         palindrom.remoteObj = JSON.parse(JSON.stringify(json));
       }
 
+      if(!palindrom.isObjectProxified) { // first time the state is being set
```

Comment:
@tomalec yes it's needed and it's not expensive. It's just disabling the callback, isn't deep cloning or doing anything like that. Only first call of `observe` is expensive.

If I don't unobserve, `onPatchApplied` will be called before `callback`. `callback` has be to be called the first thing to `propagateDomBind` in `palindrom-client`.